torchvision/models/resnet.py

```diff
@@ -34,6 +34,7 @@ def conv1x1(in_planes, out_planes, stride=1):
 
 class BasicBlock(nn.Module):
     expansion = 1
+    __constants__ = ['downsample']
```

Comment:
Can you explain a bit why this is a `constant`? `downsample` has learnable parameters.

Reply:
Ok, I think I got it. I think I prefer the `__constants__` in this case, even though `Optional[Module]` would be nicer.